__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  it("responds with an array with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = res.body.topics;
+        expect(Array.isArray(topics)).toEqual(true);
```

Comment:
Your assertion on line 17 is not needed because you're using a forEach on topics on line 19. If topics was not an array line 19 would fail. You could take line 17 out